Networker/Server/Abstractions/ITcpConnection.cs

```diff
@@ -5,5 +5,10 @@ namespace Networker.Server.Abstractions
     public interface ITcpConnection
     {
         Socket Socket { get; set; }
+
+        /// <summary>
+        /// Store a custom object on this connection.
+        /// </summary>
+        object UserTag { get; set; }
```

Comment:
Can you rename this to something like ConnectionTag or ConnectionIdentifier because the connection might not be a "user".